lnrpc/routerrpc/router.proto

```diff
@@ -502,6 +502,36 @@ message MissionControlConfig {
     before we raise the failure amount.
     */
     uint64 minimum_failure_relax_interval = 5;
+
+    enum ProbabilityModel {
+        UNKNOWN = 0;
+        APRIORI = 1;
+        BIMODAL = 2;
+    }
+
+    /*
+    ProbabilityModel defines which probability estimator should be used in
+    pathfinding.
+    */
+    ProbabilityModel Model = 6;
+
+    /*
+    BimodalNodeWeight defines how strongly non-routed channels should be taken
+    into account for probability estimation. Valid values are in [0,1].
+    */
+    float BimodalNodeWeight = 7;
```

Comment:
We can potentially fold these into a `oneof` that'll be populated based on the `ProbabilityModel` enum above.